maintainers/scripts/update.nix

```diff
@@ -8,22 +8,24 @@
 { package ? null
 , maintainer ? null
 , predicate ? null
+, get-script ? pkg: pkg.updateScript or null
```

Comment:
@jtojnar would you mind taking a look at the update.nix changes?

I've used it to implement `update-dotnet-lockfiles.nix`, which is quite similar but uses a different attribute.  This way it benefits from parallel execution, log aggregation and all the other good stuff the python script does.

I was wondering if aliases should just be unconditionally disabled?

Reply:
Yeah, I would just disable aliases unconditionally.

But why not just rename the `fetch-deps` to `updateScript`?

Reply:
`fetch-deps` just does lockfile generation.  It's often called from `updateScript`, but can be run on its own.  Quite a few packages only have `fetch-deps`.

I've been thinking it would be nice to have some sort of abstraction of lockfile generation, but I'm not exactly sure where to start.  I noticed that `nix-update` has `--generate-lockfile`, but it looks like that might only understand cargo lockfiles.

Reply:
> Yeah, I would just disable aliases unconditionally.

this is done now